### DISCUSSION TITLE
Unable to install under Ubuntu Studio 24.04 : dependency problem

### DISCUSSION THREAD

**Discussion by essemo** [GrandOrgue/grandorgue#1996] [Help]
Unable to install on Ubuntu Studio 24.04: dependency problem
Hello,
I am extremely grateful to the developers of GO that I appreciate and use almost every day!
Here is my difficulty: after complete installation of Ubuntu Studio 24.04, the installation of the GO .deb package is impossible due to a dependency problem.

After installing via Terminal (sudo apt-get install /.../grandorgue_3.15.0-1_amd64.deb)
I get the following error:
grandorgue: Depends: libwxgtk3.0-gtk3-0v5 (>= 3.0.4+dfsg) but it is not installable
Depends: libwxgtk3.0-gtk3-0v5 (>= 3.0.4+dfsg-10~) but it is not installable

I specify that libwxgtk3.2 is properly installed,
and that GO 3.13.1 can be properly installed via Discover.
The problem occurs when installing version 3.14 and 3.15.

Do you have a solution please?

The .Appimage version works well but I would prefer a complete installation.

(My configuration:
Intel® Core™ i7-4790 CPU @ 3.60GHz
31.2 GB RAM
UBUNTU STUDIO 24.04)

Thank you very much for your help...
With my gratitude and encouragement!
Best regards
JBG

**Comment by kerkovits** [comment]
You should install grandorgue-wx32 instead of grandorgue.

**Reply by essemo** [reply]
Thank you very much for your help: everything is fine thanks to you!
I admit to being a bit of a beginner: I didn't know the difference in this wx32 version at all...
Without wanting to waste your time, I would be happy to know why, when going from Ub Studio 22.04 to 24.04, you have to change version?
In any case, I thank you warmly.
Best regards!

**Reply by larspalo** [reply]
> Without wanting to waste your time, I would be happy to know why, when going from Ub Studio 22.04 to 24.04, you have to change version?

Because a 22.04 distribution has wxWidgets library version 3.0 in the repository while the 24.04 has wxWidgets 3.2. The available library version must match the version the program itself was compiled with.
